src/api_routes_http.rs

```diff
@@ -159,285 +161,281 @@ use lemmy_apub::api::{
+          .route("/unread_count", get().to(unread_count))
+          .route("/verify_email", post().to(verify_email))
+          .route("/leave_admin", post().to(leave_admin))
+          .route("/totp/generate", post().to(generate_totp_secret))
+          .route("/totp/update", post().to(update_totp))
+          .route("/list_logins", get().to(list_logins))
+          .route("/validate_auth", get().to(validate_auth)),
       )
       // Admin Actions
       .service(
-        web::scope("/admin")
+        scope("/admin")
           .wrap(rate_limit.message())
-          .route("/add", web::post().to(add_admin))
+          .route("/add", post().to(add_admin))
           .route(
             "/registration_application/count",
-            web::get().to(get_unread_registration_application_count),
+            get().to(get_unread_registration_application_count),
           )
           .route(
             "/registration_application/list",
-            web::get().to(list_registration_applications),
+            get().to(list_registration_applications),
           )
           .route(
             "/registration_application/approve",
-            web::put().to(approve_registration_application),
+            put().to(approve_registration_application),
           )
           .route(
             "/registration_application",
-            web::get().to(get_registration_application),
+            get().to(get_registration_application),
           )
-          .route("/list_all_media", web::get().to(list_all_media))
+          .route("/list_all_media", get().to(list_all_media))
           .service(
-            web::scope("/purge")
-              .route("/person", web::post().to(purge_person))
-              .route("/community", web::post().to(purge_community))
-              .route("/post", web::post().to(purge_post))
-              .route("/comment", web::post().to(purge_comment)),
+            scope("/purge")
+              .route("/person", post().to(purge_person))
+              .route("/community", post().to(purge_community))
+              .route("/post", post().to(purge_post))
+              .route("/comment", post().to(purge_comment)),
           )
           .service(
-            web::scope("/tagline")
+            scope("/tagline")
               .wrap(rate_limit.message())
-              .route("", web::post().to(create_tagline))
-              .route("", web::put().to(update_tagline))
-              .route("/delete", web::post().to(delete_tagline))
-              .route("/list", web::get().to(list_taglines)),
-          ),
+              .route("", post().to(create_tagline))
+              .route("", put().to(update_tagline))
+              .route("/delete", post().to(delete_tagline))
+              .route("/list", get().to(list_taglines)),
+          )
+          .route("block_instance", post().to(admin_block_instance))
+          .route("allow_instance", post().to(admin_allow_instance)),
```

Comment:
I removed useless the `web::` prefix all over this file, only real change is here.